jenkins-dsl/cassandra_pipeline.groovy

```diff
@@ -40,7 +40,7 @@ pipeline {
             stage('JVM DTests') {
               steps {
                   warnError('Tests unstable') {
-                    build job: "${env.JOB_NAME}-test-jvm-dtest-forking", parameters: [string(name: 'REPO', value: params.REPO), string(name: 'BRANCH', value: params.BRANCH)]
+                    build job: "${env.JOB_NAME}-jvm-dtest", parameters: [string(name: 'REPO', value: params.REPO), string(name: 'BRANCH', value: params.BRANCH)]
```

Comment:
changing the name is not so easy, because it also needs to be changed [here](https://github.com/apache/cassandra/blob/trunk/.jenkins/Jenkinsfile#L88)

Reply:
https://github.com/apache/cassandra/pull/471

checked 2.2 and the cherry pick failed, so will add PR per branch which doesn't clean apply.

Reply:
https://github.com/apache/cassandra/pull/472

tested that 3.11's patch cherry picks into 3.0 and 2.2 cleanly; for some reason trunk -> 3.11 doesn't.

Reply:
Could we just keep the name suffix as `-test-jvm-dtest-forking` for now, and come back to changing it later on?

(I suspect there will be other changes that involve updating both repositories, over multiple branches, later on. and would rather just avoid it for now :( apologises for the confusion)

Reply:
sure, will do.